azurerm/resource_arm_data_lake_store_test.go

```diff
@@ -0,0 +1,208 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMDataLakeStore_payasyougo(t *testing.T) {
+	resourceName := "azurerm_data_lake_store.test"
+	ri := acctest.RandIntRange(1, 999999)
```

Comment:
Could we use `acctest.RandInt()` for the tests here?

Reply:
there is a limitation on the name for an ADLS account which has to be between 3 to 24 characters. I wanted to restrict the length of the name to within that limit. RandInt() generates a larger string if I am not mistaken.